README.md

```diff
@@ -40,11 +37,12 @@ Generate API SDK clients for your favorite language including JavaScript, TypeSc
 
 ![generate-api-client](./media/generate-api-client.png)
 
-## Notice/Known Issues
+### Export API specification

```

Comment:
I think it is better to move this into the 'Explore' section, and no need for `## Export API specification`

Reply:
I merged it with Consume section. Please check. It better fits into Consume section as the portal also works that way.